src/clj/web/stats.clj

```diff
@@ -174,8 +175,26 @@
                 :runner.agenda-points (get-in @state [:runner :agenda-point])}
      :history (:history @state)}))
 
+(defn handle-angelarena-stats
+  [db {:keys [ending-players original-players state format] :as game}]
+  (try
+    (doall
+      (for [player original-players]
```

Comment:
If you're not using the return value of `for`, better to use `doseq` which has the same api but isn't lazy and returns `nil`.